src/plinder/data/utils/annotations/interaction_utils.py

```diff
@@ -75,6 +76,37 @@ def read_mmcif_file(mmcif_filename: Path) -> PDBxFile:
     return pdbx_file
 
 
+def get_symmetry_mate_contacts(
+    mmcif_filename: Path, contact_threshold: float = 5.0
+) -> dict[tuple[str, int], dict[tuple[str, int], set[int]]]:
+    """
+    Get all contacts within a given threshold between residues which are not in the same asymmetric unit (symmetry mates)
+    """
+    cif = gemmi.read_structure(mmcif_filename.__str__(), merge_chain_parts=False)
+    cif.setup_entities()
+    ns = gemmi.NeighborSearch(cif[0], cif.cell, contact_threshold).populate(
+        include_h=False
```

Comment:
@OleinikovasV okay to ignore crystal contacts with hydrogens?

Reply:
yes, this makes sense to me only to consider heavy atoms for a given cutoff